docs/tcp/traffic-policy/index.mdx

```diff
@@ -23,8 +23,8 @@ import TrafficPolicyWarning from "/shared/warnings/traffic-policy-preview.mdx";
 ## Overview
 
 This module allows you to assign a policy to your endpoints containing a series
```

Comment:
Can we sneak "phase" in here somehow?

"This module enables you to assign a policy to your endpoints by defining a set of rules for the `on_tcp_connect` traffic management phase. These rules allow you to influence and control traffic to your upstream service."

Reply:
yep, i'll use that instead